src/app/core/user-module/user.service.ts

```diff
@@ -146,6 +137,24 @@ export class UserService {
       .toPromise();
   }
 
+  /**
+   * Validate a user's captcha token
+   *
+   * @param {string} token the token to verify
+   * @returns an object with the result if fail, or null if true.
+   */
+  validateToken(token: string) {
```

Comment:
Can we call this validate captcha instead of validate token because validate token is a separate route that we have in auth